src/main/windows/callsWidgetWindow.ts

```diff
@@ -250,6 +246,16 @@ export default class CallsWidgetWindow extends EventEmitter {
 
         // Let the webContentsEventManager handle links that try to open a new window
         webContentsEventManager.addWebContentsEventListeners(this.popOut.webContents);
+
+        // Need to capture and handle redirects for security.
+        this.popOut.webContents.on('will-redirect', this.onWillRedirect);
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    private onWillRedirect = (event: Event, url: string) => {
```

Comment:
Is this needed? Can't we just accept the event alone?

Reply:
We just want to make sure people don't find a way to force the popout to go to a different URL.